### PR TITLE
[DA-1860] Genomic tool for backfilling AW1 file processed IDs

### PR DESCRIPTION
**Pull request by j-kanuch**
This PR creates a tool to back fill `genomic_set_member.aw1_file_processed_id` from a provided list of package IDs. AW1 files are matched to a `genomic_set_member` record from the `genomic_set_member.package_id` field and the package id string in the AW1 file name.
Additionally, more `GenomicManifestType` enum entries were added for previously missing manifest types.

### CODE REVIEW

File: rdr_service/tools/tool_libs/genomic_utils.py [hunk 1]
```diff
@@ -1208,6 +1309,12 @@ def run():
     collection_tube_parser.add_argument("--sample-override", help="for testing",
                                         default=False, action="store_true")  # noqa
 
+    # Backfill tool for genomic_set_member.aw1_file_processed_ID
+    backfill_file_processed_id_parser = subparser.add_parser("file-processed-id-backfill")
+    backfill_file_processed_id_parser.add_argument("--csv", help="A CSV file with the package_ids to backfill",
+                                        default=None, required=True)
+    backfill_file_processed_id_parser.add_argument("--dryrun",
+                                                   help="for testing", default=False, action="store_true")  # noqa
```
Comment (kenny-skaggs) [inline rdr_service/tools/tool_libs/genomic_utils.py]: Everything looks good. Just one possible suggestion: would it make sense to add the "--dry-run" argument on the main parser (`parser`) so it doesn't have to be defined for each subparser?

Comment (j-kanuch) [reply rdr_service/tools/tool_libs/genomic_utils.py]: Good suggestion. I've moved it to the main parser and removed each instance from the subparsers.

File: rdr_service/tools/tool_libs/genomic_utils.py [hunk 2]
```diff
@@ -1194,20 +1291,18 @@ def run():
                                        default=None, required=False)
     process_runner_parser.add_argument("--csv", help="A file specifying multiple manifests to process",
                                        default=None, required=False)
-    process_runner_parser.add_argument("--dryrun", help="for testing", default=False, action="store_true")  # noqa
-
-    # Backfill GenomicFileProcessed UploadDate
-    upload_date_parser = subparser.add_parser("backfill-upload-date")
```
Comment (kenny-skaggs) [inline rdr_service/tools/tool_libs/genomic_utils.py:1200]: Is this subparser still needed to trigger the backfill-upload-date if-block later?

Comment (j-kanuch) [reply rdr_service/tools/tool_libs/genomic_utils.py:1200]: It looks like this line is still needed. I'll add back in.

**Review by kenny-skaggs — APPROVED** [review]

**Pull request merged**